src/layout/Footer.js

```diff
@@ -68,21 +68,23 @@ class Footer extends Component {
       <FooterContainer>
         <FooterInner>
           <H2>Get in touch</H2>
-          <a href="mailto:hello@kickback.events">hello@kickback.events</a>
+          <p>
+            <a style={{color:'white'}}href="mailto:hello@kickback.events">hello@kickback.events</a>
```

Comment:
Need space before href. Also, why aren't we using styled-components for this? See other files for examples of how to style elements, or e.g. `Row` element below.